### PR TITLE
Add Icon for Unity

### PR DESCRIPTION
**Pull request by NoSilver2018**
**Issue:**
Add Icon(third try)

### Checklist
  - [ o ] I optimized the icon with SVGO or SVGOMG
  - [ o ] The SVG `viewbox` is `0 0 24 24`

### Description

- Because unity's default icon color is black, I set the hex value like this.
- Following contributer's advise, I used this [homepage](https://www.iloveimg.com/) for resizing.
- OS: Mac OSX High Sierra

### CODE REVIEW

**Comment by NoSilver2018** [comment]
Sorry for this repetitive Pull Request! I would like to try to find the failing reason. I'll check the detail of Travis CI Build!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2300,6 +2300,11 @@
             "hex": "6FDA44",
             "source": "https://www.upwork.com/press/"
         },
+	{
```
Comment (birjolaxew) [inline _data/simple-icons.json]: The indentation is a bit off here - looks like you're using tabs while the rest of the file is using 8 spaces. We'd prefer to keep the indentation consistent throughout.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -2300,6 +2300,11 @@
...
+		"title": "Unity",
+		"hex": "000000",
+		"source": "https://vectors.pro/unity-3d-vector-icon-logo/"
```
Comment (birjolaxew) [inline _data/simple-icons.json]: We'd prefer an official source if we can get one. In this case https://unity3d.com/ actually includes [an SVG](https://unity3d.com/profiles/unity3d/themes/unity/images/ui/ui/unity-logo-white.svg) in their navbar - all that's left to do is crop, resize and annotate it.

File: icons/unity.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img"><title>Unity icon</title><path d="M15.4102 12l4.0273-6.9492L21.3789 12l-1.9414 6.9492zm-1.961 1.129l4.0235 6.9491-7.0079-1.7968-5.0664-5.1524zm4.0235-9.2071l-4.0235 6.9492H5.3984l5.0664-5.1524zm5.746 5.7148L20.7658.5l-9.168 2.4492-1.3555 2.3828-2.754-.0195L.7813 12l6.707 6.6875 2.754-.0195 1.3594 2.3828 9.164 2.4492 2.4532-9.1328L21.8241 12zm0 0"/></svg>
```
Comment (birjolaxew) [inline icons/unity.svg]: We'd like the icon to fill its 24x24 viewbox as much as possible. Looks like you have a bit of whitespace around the icon itself here.

File: _data/simple-icons.json (hunk 2 above)
Comment (NoSilver2018) [reply _data/simple-icons.json]: Thank you for your kind anwer! I've modified all thing you mentioned directly. :D!

**Comment by pzygielo** [comment]
Looks that after cbfb586b2405d12e96777cc15cf40a12adcd5d02 `icons/unity.svg` is not valid xml.
@birjolaxew - could travis build validate xml for icons? Seems strange it passed with this case.

**Comment by birjolaxew** [comment]
@pzygielo The [linter](https://github.com/birjolaxew/svglint) uses the very forgiving htmlparser2 parser - if you would be willing to contribute over at [birjolaxew/svglint](https://github.com/birjolaxew/svglint) to move to a more strict XML parser, that'd be very much appreciated!

File: _data/simple-icons.json (hunk 1 above)
Comment (NoSilver2018) [reply _data/simple-icons.json]: I edited this part following your advise!

File: icons/unity.svg [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img"><title>Unity icon</title>
```
Comment (birjolaxew) [inline icons/unity.svg]: The SVG seems to not fill its viewbox entirely - there is a bit of whitespace around it (especially to the left of it).

The SVG itself should also be on one line. There's currently a newline after `</title>`

**Pull request closed without merging**